### PR TITLE
Added stripe donations

### PR DESCRIPTION
**Pull request by kayone**

### CODE REVIEW

File: js/donate.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+(function($) {
+  var donationAmount = $('#donation-amount');
+  var currency = 'usd';
+  var key = 'pk_test_aKr8O3jA8gVWltZ9dxzEl8qF';
```
Comment (kayone) [inline js/donate.js:4]: need to update this to prod

File: js/donate.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  var chargeApi = 'http://localhost:8089/v1/payments/stripe/charge';
```
Comment (kayone) [inline js/donate.js]: same with this,

File: js/donate.js (hunk 1 above)
Comment (markus101) [inline js/donate.js:4]: Any security risk with this key?

File: js/donate.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      type: "POST",
+      url: chargeApi,
+      data: options,
+      dataType: 'json'
+    }).done(function(){
+      debugger;
+    });
+  }
+
+  donationAmount.on('keyup', function(e){
```
Comment (markus101) [inline js/donate.js:24]: What if they paste a value in? Maybe handle `change` as well?

File: js/donate.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  function createCharge(options) {
+    $.ajax({
+      type: "POST",
+      url: chargeApi,
+      data: options,
+      dataType: 'json'
+    }).done(function(){
+      debugger;
```
Comment (markus101) [inline js/donate.js]: For testing?

File: js/donate.js (hunk 1 above)
Comment (kayone) [inline js/donate.js:4]: no this is the public key,

File: js/donate.js (hunk 4 above)
Comment (kayone) [inline js/donate.js]: arg, for breaking IE probably.

**Pull request merged**